docs/kyma/docs/040-cr-installation.md

```diff
@@ -26,14 +28,24 @@ metadata:
 spec:
   version: "1.0.0"
   url: "https://sample.url.com/kyma_release.tar.gz"
+  components: 
+    - name: "remote-environments"
+      namespace: "kyma-integration"
+      release: "hmc-default"
 ```
 
+## Custom resource parameters
+
 This table lists all the possible parameters of a given resource together with their descriptions:
 
 | Field   |      Mandatory?      |  Description |
 |:----------:|:-------------:|:------|
 | **metadata.name** | **YES** | Specifies the name of the CR. |
-| **metadata.labels.action** | **YES** | Defines the behavior of the Kyma installer. Available options are `install` and `uninstall`. |
+| **metadata.labels.action** | **YES** | Defines the behavior of the Kyma installer. Available options are `install`, `update`, and `uninstall`. |
 | **metadata.finalizers** | **NO** | Protects the CR from deletion. Read [this](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers) Kubernetes document to learn more about finalizers. |
 | **spec.version** | **NO** | When manually installing Kyma on a cluster, specify any valid [SemVer](https://semver.org/) notation string. |
 | **spec.url** | **YES** | Specifies the location of the Kyma sources `tar.gz` package. For example, for the `master` branch of Kyma, the address is `https://github.com/kyma-project/kyma/archive/master.tar.gz` |
+| **spec.components** | **YES** | Lists which components of Helm chart components to `install` or `update`. |
+| **spec.components.name** | **YES** | Specifies the name of the component which is the same as the name of the component subdirectory in the `resources` directory. |
+| **spec.components.namespace** | **YES** | Defines the Namespace in which you want the Installer to `install`, or `update` the component. |
+| **spec.components.release** | **NO** | Provides the name of the Helm release in case it should differ from the component name. |
```

Comment:
I would rephrase it like: "Provides the name of the Helm release. Defaults to component name."

Reply:
"Provides the name of the Helm release. The default one is the component name."